app/src/main/java/com/viscouspot/gitsync/GitSyncAccessibilityService.kt

```diff
@@ -25,7 +25,7 @@ class GitSyncAccessibilityService: AccessibilityService() {
     override fun onAccessibilityEvent(event: AccessibilityEvent?) {
         val packageNames = settingsManager.getApplicationPackages()
 
-        if (!(settingsManager.getApplicationObserverEnabled() && packageNames.isNotEmpty() && (settingsManager.getSyncOnAppClosed() || settingsManager.getSyncOnAppOpened()))) return
```

Comment:
getApplicationObserverEnabled now acts as a dropdown switch only and is used only to store the state of the dropdown